### PR TITLE
tag unstable functionality

### PR DESCRIPTION
**Pull request by DenisBiryukov91**
tag unstable functionality

### CODE REVIEW

File: zenoh/__init__.py [hunk 1]
```diff
@@ -17,3 +17,7 @@
     from . import ext
 except ImportError:
     pass
+
+
+def _unstable(item):
```
Comment (wyfo) [inline zenoh/__init__.py:22]: This function is not needed in the code. It's fine to have private items in stubs that don't correspond to a real exported item.

File: zenoh/__init__.pyi [hunk 2]
```diff
@@ -349,6 +352,7 @@ class KeyExpr:
 _IntoKeyExpr = KeyExpr | str
 
 @final
+@_unstable
```
Comment (wyfo) [inline zenoh/__init__.pyi:355]: I would rather put `@_unstable` above the `@final`, but it's a matter of taste.

**Review by wyfo — CHANGES_REQUESTED** [review]

File: zenoh/__init__.py (hunk 1 above)
Comment (DenisBiryukov91) [reply zenoh/__init__.py:22]: it is needed for stubs_check.py test to pass.

File: zenoh/__init__.pyi (hunk 2 above)
Comment (DenisBiryukov91) [reply zenoh/__init__.pyi:355]: fixed

File: zenoh/__init__.py (hunk 1 above)
Comment (wyfo) [reply zenoh/__init__.py:22]: Then modify `stubs_check.py`

Comment (DenisBiryukov91) [reply zenoh/__init__.py]: done

**Review by wyfo — APPROVED** [review]

**Pull request merged**